src/sysen2/mldev.108

```diff
@@ -166,22 +194,45 @@ notits:	;; nothing came of it, fall through
 			.SUSET [.RUNAME,,JBCFN2]
 			JRST (A) ]
 		MOVEI A,'ML		;USE THE MATHLAB LINE PRINTER
-		JRST GOTHST ]
+		JRST GOTHSX ]
 ;HOST NOT RECOGNIZED
 NSD:	MOVSI A,%ENSDV
 	JRST NOGO2
 
+hhhddd:				; get device from last 3 characters
+	hrlz c,a		; get device and left justify
+	movem c,jbcdev		; save remote device
+	hllzs a			; get hostname
+	jrst gothst		; process host
```

Comment:
Also here you haven't tested if it is an ITS name. Wouldn't it be "smoother" to change the "jumpn c,hhhddd" above to a "skipn c" + "move c,dskdev"?

Reply:
Not sure. You are reviewing old code.  Please take a look at the latest.